test/Combinatorics/runtests.jl

```diff
@@ -5,4 +5,5 @@ include("SimplicialComplexes.jl")
 include("Matroids/Matroids.jl")
 include("Matroids/Chow_Ring.jl")
 include("Graph.jl")
-include("Matroids/matroid_strata_grassmannian.jl")
```

Comment:
The `runtests.jl` files are basically gone, we now basically just execute all the `*.jl` files in subdirectories of `test/`. So resolving the merge conflict should hopefully just require deleting this file/change.

Reply:
I got rid of the `matroid_strata_grassmannian.jl` file, and commented everything in the `runtests.jl` file. (I guess we could also just delete it)

Reply:
@dcorey2814 that's the opposite of what is needed: the runtests.jl must be deleted. And your new test file of course should be kept

Reply:
Ok, I removed `runtests.jl`. And yes, I believe that `matroid_strata_grassmannian.jl` is now obsolete.